index.html

```diff
@@ -1392,9 +1392,9 @@ <h2>Core Vocabulary Definitions</h2>
 		  It is considered to be good practice that each <code>observeproperty</code> has a corresponding
 		  <code>unobserveproperty</code> unless the protocol supports implicit unsubscription mechanisms
 		  (e.g., heartbeat to detect connection loss).</p><p class="note">
-		  The observation mechanism depends on the underlying protocol or sub-protocol.  It is not guaranteed that the latest <a>Property</a> value will be provided when 
-          the subscription is initiated. To observe the latest <a>Property</a> value, it is recommended to first read the <a>Property</a> (<code>readproperty</code>) 
-          and then perform a <a>Property</a> subscription (<code>observeproperty</code>). 
+		  The observation mechanism depends on the underlying protocol or sub-protocol. Having said that, it is not 
+          guaranteed that the current Property value will be provided once the subscription is initiated. Hence, it may be 
+          necessary to read the current Property value before/after the subscription to get a first value. 
```

Comment:
Shall we uses `<a>Property</a>` instead of Property as you did before in the HTML code? Either way is fine by me.

Reply:
done